src/domain/permissions.js

```diff
@@ -1,70 +1,35 @@
 import { ROLE } from './constants.js';
 
-const ADMIN = 'admin';
-const EDIT_DOC = 'edit-doc';
-const VIEW_DOCS = 'view-docs';
-const EDIT_FILE = 'edit-file';
-const OWN_ROOMS = 'own-rooms';
-const REVIEW_DOC = 'review-doc';
-const VERIFY_DOC = 'verify-doc';
-const VIEW_FILES = 'view-files';
-const JOIN_ROOMS = 'join-rooms';
-const CREATE_FILE = 'create-file';
-const ARCHIVE_DOC = 'archive-doc';
-const MIGRATE_DATA = 'migrate-data';
-const SEARCH_USERS = 'search-users';
+const VIEW_USERS = 'view-users';
 const MANAGE_USERS = 'manage-users';
-const MANAGE_BATCHES = 'manage-batches';
-const SEE_USER_EMAIL = 'see-user-email';
-const MANAGE_CONTENT = 'manage-content';
-const PROTECT_ANY_DOC = 'protect-any-doc';
-const PROTECT_OWN_DOC = 'protect-own-doc';
-const MANAGE_SETTINGS = 'manage-settings';
-const DELETE_OWN_FILES = 'delete-own-files';
-const HARD_DELETE_SECTION = 'hard-delete-section';
-const DELETE_FOREIGN_ROOMS = 'delete-foreign-rooms';
-const MANAGE_STORAGE_PLANS = 'manage-storage-plans';
-const DELETE_ANY_STORAGE_FILE = 'delete-storage-file';
-const RESTORE_DOC_REVISIONS = 'restore-doc-revisions';
-const CREATE_DOCUMENT_COMMENTS = 'create-document-comments';
-const MANAGE_DOCUMENT_COMMENTS = 'manage-document-comments';
-const AUTORIZE_ROOMS_RESOURCES = 'authorize-room-resources';
-const MANAGE_ACCREDITED_EDITORS = 'manage-accredited-editors';
+const MANAGE_SETUP = 'manage-setup';
+const CREATE_CONTENT = 'create-content';
+const ACCESS_STORAGE = 'access-storage';
+const BATCH_PROCESS_DATA = 'batch-process-data';
+const MANAGE_PUBLIC_CONTENT = 'manage-public-content';
+const PROTECT_OWN_PUBLIC_CONTENT = 'protect-own-public-content';
+const DELETE_OWN_PRIVATE_CONTENT = 'delete-own-private-content';
+const DELETE_ANY_PRIVATE_CONTENT = 'delete-any-private-content';
+const MANAGE_PROTECTORS_OF_OWN_PUBLIC_CONTENT = 'manage-protectors-of-own-public-content';
 
 const userPermissions = [
-  EDIT_DOC,
-  VIEW_DOCS,
-  EDIT_FILE,
-  VIEW_FILES,
-  DELETE_OWN_FILES,
-  CREATE_FILE,
-  OWN_ROOMS,
-  AUTORIZE_ROOMS_RESOURCES,
-  JOIN_ROOMS,
-  CREATE_DOCUMENT_COMMENTS
+  CREATE_CONTENT,
+  ACCESS_STORAGE,
```

Comment:
Given the very nice logic this PR introduces (namely that the permission names now always clearly describe some action that goes beyond any action that any anonymous user could perform anyways), the name of this permission could be `BROWSE_STORAGE` or `SEARCH_STORAGE`, because anyone can access it anyways (except for room resources). And if you search for usages of this permission, you find it is always used in the scenario of actively searching a file by some means. Except for this one instance: in the `/api/v1/rooms/:roomId/authorize-resources-access` endpoint. But honestly, we could remove it from there anyways, because the actually deciding factor in this endpoint is the user's room membership and therefore access to specific a room resource, not storage access in general, so logically it is a little misleading anyways. What do you think?

Reply:
I completely agree and the authorization endpoint was the only reason why I tried to come up with a permission name to fit this case as well.
So then we can go with `BROWSE_STORAGE` on all endpoints, except for the authorization one, where we can switch to `needsAuthentication()`, right?

Reply:
Exactly! :+1: :+1: :+1:

Reply:
Great, changed it just so.